src/main/java/duke/ByeCommand.java

```diff
@@ -1,5 +1,8 @@
 package duke;
 
+/**
+ * Duke Command for exiting the program.
+ */
```

Comment:
I notice you have created java docs for all of your classes. Keep up the good work! 👍